### ISSUE TITLE
Cannot find module 'query-string' from 'route-utils.js'

### ISSUE THREAD

**Issue by brandongregoryscott** [AndcultureCode/AndcultureCode.JavaScript.Core#14]
Wanted to update `andculturecode-javascript-react` to use the latest version of this package, 0.0.8, and the tests for that project are now bombing out. Looks like `query-string` will need to be moved to a prod dependency in `package.json`. Again, not direly urgent - these packages are very much a work-in-progress. I'll backtrack on the version upgrade for now, since 0.0.7 seems to be running without issue.

```
 FAIL  src/utilities/route-utils.test.ts
  ● Test suite failed to run

    Cannot find module 'query-string' from 'route-utils.js'

    Require stack:
      node_modules/andculturecode-javascript-core/dist/utilities/route-utils.js
      node_modules/andculturecode-javascript-core/dist/index.js
      src/utilities/route-utils.ts
      src/utilities/route-utils.test.ts

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:299:11)
      at Object.<anonymous> (node_modules/andculturecode-javascript-core/src/utilities/route-utils.ts:1:1)
```

**Comment by wintondeshong** [comment]
Bah! Oversight on my part. I'll resolve this in my current work.

**Comment by wintondeshong** [comment]
Fixed with release v0.1.0 https://github.com/AndcultureCode/AndcultureCode.JavaScript.Core/releases/tag/0.1.0
